AK/BinaryBufferWriter.h

```diff
@@ -19,7 +19,8 @@ class BinaryBufferWriter {
     }
 
     template<typename T>
-    requires(AK::Detail::IsTriviallyConstructible<T>) T& append_structure()
+        requires(AK::Detail::IsTriviallyConstructible<T>)
+    T& append_structure()
```

Comment:
Can we configure this? "requires on the same line as the rest of the function decl"?

Reply:
Reformatted with

```
RequiresClausePosition: WithFollowing
IndentRequiresClause: false
```

Still does a bunch of changes tho.

Reply:
Seems like the best of the available options, so :shrug: